### DISCUSSION TITLE
Call to all contributors to sign the newly added Code Licence Agreement

### DISCUSSION THREAD

**Discussion by theRookieCoder** [gorilla-devs/ferium#161] [Announcements]
@SolidTux @sidit77 @Ember-Ember @obj-obj @KyleUltimate @A-noximous @douira @4JX @tuxinal

The Gorilla Devs organisation has decided to add a Code Licence Agreement (CLA) to all of its repositories. The purpose of a CLA is to give ownership of the code you write, to us the @gorilla-devs organisation. This allows us to have more control over the project, e.g. change the licence.

_I would yet again like to reiterate that by signing this CLA, **you give us ownership of the code you have written or will write to us**._ Don't worry, you will still be considered the author of the code.

I of course urge you to read the full CLA before signing. Signing it once will allow you to write code across all of @gorilla-devs's repositories. Feel free to ask any doubts!

https://cla-assistant.io/gorilla-devs/ferium

**Comment by JustSimplyKyle** [comment]
Hmm, where can I check the CLA. I can't seem to find it

**Reply by theRookieCoder** [reply]
I forgor 💀

**Reply by JustSimplyKyle** [reply]
I'm got stuck here
![image](https://user-images.githubusercontent.com/68589851/176699966-eb2e5a0e-3195-43b0-9760-2219ae84a3a9.png)

**Reply by theRookieCoder** [reply]
https://github.com/gorilla-devs/ferium/discussions/161#discussioncomment-3056893

**Comment by theRookieCoder** [comment]
For some reason the website doesn't work on Firefox :(

It takes forever to load, so I suggest you temporarily use another browser

**Reply by JustSimplyKyle** [reply]
Ok, finished!

**Reply by tuxinal** [reply]
Done

**Reply by blarfoon** [reply]
cla-assistant absolutely sucks tbh but it's the best in town afaik :(

**Reply by sidit77** [reply]
done

**Comment by obj-obj** [comment]
I'm pretty sure my contributions are adding like 3 lines to Cargo.toml but I'll sign it lmao

**Comment by A-noximous** [comment]
My only contribution was changing a few lines in README.MD, but I have signed. Good luck gorilla-devs 👍.

**Comment by 4JX** [comment]
Apologies for the wait, signed as well.

**Reply by theRookieCoder** [reply]
No problem! It's not urgent (unless you're going to contribute code _now_)
